### ISSUE TITLE
nvm use not work

### ISSUE THREAD

**Issue by AveVlad** [coreybutler/nvm-windows#18]
video -> http://www.youtube.com/watch?v=L9rSo6e32yM

```
C:\Windows\system32>nvm

Running version 1.0.6.

Usage:

  nvm arch                     : Show if node is running in 32 or 64 bit mode.
  nvm install <version> [arch] : The version can be a node.js version or "latest" for the latest stable version.
                                 Optionally specify whether to install the 32 or 64 bit version (defaults to system arch).
                                 Set [arch] to "all" to install 32 AND 64 bit versions.
  nvm list [available]         : List the node.js installations. Type "available" at the end to see what can be installed. Aliased as ls.
  nvm on                       : Enable node.js version management.
  nvm off                      : Disable node.js version management.
  nvm proxy [url]              : Set a proxy to use for downloads. Leave [url] b lank to see the current proxy.
                                 Set [url] to "none" to remove the proxy.
  nvm uninstall <version>      : The version must be a specific version.
  nvm use [version] [arch]     : Switch to use the specified version. Optionally specify 32/64bit architecture.
  nvm use <arch> will continue using the selected version, but switch to 32/64 bit mode.
  nvm root [path]              : Set the directory where nvm should store different versions of node.js. If <path> is not set, the current root will bedisplayed.
  nvm version                  : Displays the current running version of nvm for Windows. Aliased as v.


C:\Windows\system32>nvm list

    0.11.14
    0.11.11
  * 0.10.32 (Currently using 64-bit executable)

C:\Windows\system32>nvm use 0.11.11
Now using node v0.11.11 (64-bit)

C:\Windows\system32>node -v
v0.10.32

C:\Windows\system32>nvm install 0.11.11 64
Version 0.11.11 is already installed.

C:\Windows\system32>nvm install 0.11.11 32
Downloading node.js version 0.11.11 (32-bit)...
C:\Windows\system32>nvm install 0.10.33
Downloading node.js version 0.10.33 (64-bit)... Complete

Could not retrieve https://raw.githubusercontent.com/coreybutler/nodedistro/mast
er/nodeversions.json.


Get https://raw.githubusercontent.com/coreybutler/nodedistro/master/nodeversions
.json: EOF
C:\Windows\system32>nvm list

    0.11.14
    0.11.11
    0.10.33
  * 0.10.32 (Currently using 64-bit executable)

C:\Windows\system32>nvm install 0.10.33 32
Downloading node.js version 0.10.33 (32-bit)... Complete
Downloading npm version 1.4.28... Complete
Installing npm v1.4.28...

Installation complete. If you want to use this version, type

nvm use 0.10.33

C:\Windows\system32>nvm list

    0.11.14
    0.11.11
    0.10.33
  * 0.10.32 (Currently using 64-bit executable)

C:\Windows\system32>node -v
v0.10.32

C:\Windows\system32>nvm use 0.10.33
Now using node v0.10.33 (64-bit)

C:\Windows\system32>node -v
v0.10.32

C:\Windows\system32>
```

**Comment by romantis** [comment]
Don't chose version before you install all versions you need.

I had the same issue.( I'm using W8.1 64.)
Here's how I solve it.
1. I tried to uninstall all versions of node installed by "nvm install 0.XX.XX". But nvm said "this version is not installed",  even though "nvm list" showed its there. So I installed it. And only after that, it uninstalled it.
2, I installed all versions of node I needed.
3. I had played switching between versions. Everything worked just fine!

**Comment by romualdr** [comment]
I guess this issue is related to #37 !

**Comment by g0t4** [comment]
FYI, if you had node installed before, delete C:\Program Files\nodejs and nvm windows will create the symlink

**Comment by moshest** [comment]
same problem for me.

**Comment by butlersrepos** [comment]
I had this issue on Win7 and **tl;dr** is that I had chosen my symlinks directory as a folder with my original installation of NodeJS and when I emptied that folderout and ran **nvm use 0.12.1** then it populated the folder and everything worked as expected for me.

    T:\Development
    λ echo %NVM_HOME%
    C:\Users\butlersrepos\AppData\Roaming\nvm

    T:\Development
    λ echo %NVM_SYMLINK%
    T:\development\nodejs

**Comment by stramel** [comment]
I got this to work by deleting the NVM_SYMLINK path that existed and running `nvm use 0.12.6` again.

**Comment by estvmachine** [comment]
Ok delete NVM_SYMLINK path that existed and running nvm use 0.x.x works for me too :+1:

**Comment by duc-nguyen-locm** [comment]
For me, there was a path to the old version in the PATH environment setting.  After removing the old from the PATH setting everything works.

**Comment by mcphelan** [comment]
I was getting this when trying to install new version

C:\WINDOWS\system32>nvm install 6.4.0

Could not retrieve https://raw.githubusercontent.com/coreybutler/nodedistro/master/nodeversions.json.

Get https://raw.githubusercontent.com/coreybutler/nodedistro/master/nodeversions.json: EOF

Tested the URL in browser and would not load with SSL exception.

Fix:
I updated my Asus router's Frimware and url was accessible in browser and nvm install command started to work.

I had suspect it was network related as command worked on works network, note home network does not use proxy.

If you are getting this exception ensure you can access the url in a browser 1st, before you start changing your Path.

**Comment by UltimateTechDeveloper** [comment]
I did what g0t4 advsed and deleted the content in the folder c:\program files\nodejs
I used the command nvm user 4.6.0 and this time it worked.
This was done on a windows 7 enterprise, 64bit

**Comment by altanai** [comment]
Same problem . Trying to update node version from 4.5.0 to 5.0.0

Removed symlink as suggested in above posts $ export NVM_SYMLINK=
Then have nvm use command $ nvm use 5.0.0
Now using node v5.0.0 (64-bit)

Checked Node version it was still 4.5 instead of 5.0.0 $ node -v
v4.5.0

Even tried uninstall , which gave an error 
$ nvm uninstall 4.5.0
Uninstalling node v4.5.0...Error removing node v4.5.0
Manually remove C:\Users\techmadmin\AppData\Roaming\nvm\v4.5.0.

Still the nvm list command shows im using node 4.5.0 insted of 5.0.0
$ nvm list
    7.6.0
    5.0.0
  * 4.5.0 (Currently using 64-bit executable)

**Comment by RoggerTipiani** [comment]
try to start with a new fresh instalation of NVM, as the documentation say, if you have a previous version of node installed on your PC, erase it and install it again using NVM.

"C:\Program Files\nodejs" and "C:\Users<user>\AppData\Roaming\npm" directories must be erased before install Node versions

**Comment by answer-huang** [comment]
Pls check your system's environment variable path, if you installed node manually, it will add path to your system's path, this may cover your nvm's path.

**Comment by coreybutler** [comment]
Closing this for the following reasons:

1) It's documented that prior editions of Node should be removed before installing NVM4W.
2) The installer now attempts to remove a prior version of Node whenever possible.
3) This issue was originally filed before the Node Foundation released a proper JSON list of available versions. The prior edition was using the hacked nodeversions.json file I maintained (which was  just scraping Node's release list for versions).
4) This thread is getting clouded with multiple issues (such as permissions). 

If anyone still experiences these issues with v1.1.7 or later, please open a new issue.
